platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMapOptions.java

```diff
@@ -566,6 +572,20 @@ public MapboxMapOptions setPrefetchesTiles(boolean enable) {
     return this;
   }
 
+  /**
+   * Enable cross source collision, defaults to true.
```

Comment:
In the docs, how about going from "cross source collision" to the somewhat more verbose but hopefully easier-to-understand "cross-source symbol collision detection"?